[2022-12-04T16:12:53.185+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-03T11:00:00+00:00 [queued]>
[2022-12-04T16:12:53.193+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-03T11:00:00+00:00 [queued]>
[2022-12-04T16:12:53.197+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:53.205+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:12:53.207+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:53.220+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): notification> on 2022-12-03 11:00:00+00:00
[2022-12-04T16:12:53.233+0000] {standard_task_runner.py:55} INFO - Started process 305 to run task
[2022-12-04T16:12:53.264+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'notification', 'scheduled__2022-12-03T11:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmpwoxyx0aq']
[2022-12-04T16:12:53.279+0000] {standard_task_runner.py:83} INFO - Job 26: Subtask notification
[2022-12-04T16:12:53.436+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.notification scheduled__2022-12-03T11:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:12:53.540+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=notification
AIRFLOW_CTX_EXECUTION_DATE=2022-12-03T11:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-03T11:00:00+00:00
[2022-12-04T16:12:53.545+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-04T16:12:53.548+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo received data at 2022-12-03']
[2022-12-04T16:12:53.580+0000] {subprocess.py:86} INFO - Output:
[2022-12-04T16:12:53.586+0000] {subprocess.py:93} INFO - received data at 2022-12-03
[2022-12-04T16:12:53.588+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-04T16:12:53.664+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=notification, execution_date=20221203T110000, start_date=20221204T161253, end_date=20221204T161253
[2022-12-04T16:12:53.729+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:12:53.758+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
